documentation/slate/source/includes/apis.md

```diff
@@ -1438,15 +1438,16 @@ access it via that kind of url: <https://api.navitia.io/v1/{a_path_to_a_resource
----------|--------------------|-----------|------------------------------------------------------------------------------------------|--------------
-nop      | from_datetime      | [iso-date-time](#iso-date-time) | The date_time from which you want the schedules                    | the current datetime
-nop      | duration           | int       | Maximum duration in seconds between from_datetime and the retrieved datetimes.           | 86400
-nop      | depth              | int       | Json response [depth](#depth)                                                            | 1
-nop      | items_per_schedule | int       | Maximum number of columns per schedule.                                                  |
-nop      | forbidden_uris[]   | id        | If you want to avoid lines, modes, networks, etc.                                        |
-nop      | data_freshness     | enum      | Define the freshness of data to use<br><ul><li>realtime</li><li>base_schedule</li></ul>  | base_schedule
-nop      | disable_geojson    | boolean   | remove geojson fields from the response                                                  | False
+Required | Name               | Type      | Description                                                                                                                | Default Value
+---------|--------------------|-----------|----------------------------------------------------------------------------------------------------------------------------|--------------
+nop      | from_datetime      | [iso-date-time](#iso-date-time) | The date_time from which you want the schedules                                                      | the current datetime
+nop      | duration           | int       | Maximum duration in seconds between from_datetime and the retrieved datetimes.                                             | 86400
+nop      | depth              | int       | Json response [depth](#depth)                                                                                              | 1
+nop      | items_per_schedule | int       | Maximum number of columns per schedule.                                                                                    |
+nop      | forbidden_uris[]   | id        | If you want to avoid lines, modes, networks, etc.                                                                          |
+nop      | data_freshness     | enum      | Define the freshness of data to use<br><ul><li>realtime</li><li>base_schedule</li></ul>                                    | base_schedule
+nop      | disable_geojson    | boolean   | remove geojson fields from the response                                                                                    | False
+nop      | direction_type     | enum      | Allow to filter the response with the route direction type property <ul><li>all</li><li>forward</li><li>backward</li></ul> | all
```

Comment:
Please add that forward filters for forward, inbound and clockwise and backward for backward, outbound and anticlockwise.
This is not straightforward, so we have to precise it.
(same on all occurences)

Reply:
is it good for you ?
![new_doc](https://user-images.githubusercontent.com/32099204/65783254-b5c89c00-e14f-11e9-9a44-154458effa19.png)